bindings/python/test/estimator/test_tle_solver.py

```
@@ -0,0 +1,195 @@
+                initial_guess=initial_state, observations=observations
+            )
+        assert "Initial guess must be a TLE or (State, B*) when fitting with B*." in str(
+            e.value
+        )
+
+    def test_estimate_tle_no_observations(
+        self,
+        tle_solver: TLESolver,
+        initial_state: State,
+    ):
+        with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)